test/index.test.js

```diff
@@ -145,3 +146,19 @@ test('env var substitution', function(t) {
 
   t.end();
 });
+
+test('env overrides for which files to read', (t) => {
+  const servEnv = __dirname + '/fixtures/serv-env';
+  process.env.SERVICE_ENV = 'foo';
+  process.env.CONFIG_SECRET_FILE = path.resolve(
+    servEnv,
+    'config.super-secret.json',
+  );
+  const config = loadConfig(servEnv);
+  t.equal(config.secret, 42);
+  t.equal(config.source, 'from-foo');
+
+  delete process.env.SERVICE_ENV;
```

Comment:
better do this in `t.teardown()` if `tape` supports it

Reply:
It doesn't.

I'd rather move everything to another test framework, but I thought I would not totally rewrite the tests as part of a total-rewrite PR. :)